src/coreclr/tools/aot/crossgen2/crossgen2.csproj

```diff
@@ -7,7 +7,7 @@
     <TargetArchitectureAppHost>$(TargetArchitecture)</TargetArchitectureAppHost>
     <TargetArchitectureAppHost Condition="'$(TargetArchitectureAppHost)'=='armel'">arm</TargetArchitectureAppHost>
 
-    <AppHostRuntimeIdentifier>$(_runtimeOS)-$(TargetArchitectureAppHost)</AppHostRuntimeIdentifier>
+    <AppHostRuntimeIdentifier>$(PackageRID)</AppHostRuntimeIdentifier>
```

Comment:
ARMEL build is broken because target architecture is not updated to arm for crossgen2.
Please fix it. Thank you.

Reply:
@clamp03 can you please open a new issue for that? @am11 do you have time to look at the regression?

Reply:
armel is another community supported platform, which gets broken time to time. For community related things to sustain in the continuous development (for the busy repo like runtime), there should be a CI leg IMO (as we added one last year for GCC build validation). I have the next batch of changes churning, and will fix it in soon. Meanwhile, you can apply a local patch and get by it (and please consider adding CI leg as it won't be the last armel breakage otherwise).

Reply:
cc @jkotas @safern regarding the armel CI protection discussion above.

Reply:
Armel fix is pushed in 55279: https://github.com/dotnet/runtime/pull/55279/files#diff-9da24614831c308827a1ae533ffea392c97638c261dd42bd0f5226baa136d16eR137-R169.

Reply:
I think it is fine to add an armel CI build that just does a traversal build. Maybe in the dev-innerloop pipeline?

Reply:
Adding armel + Tizen CI build leg is tracked by https://github.com/dotnet/runtime/issues/2394

As @safern said, we are open to adding CI build legs to protect community supported configurations. We expect the community to set them up. The steps for how to do that are in https://github.com/dotnet/runtime/issues/2394#issuecomment-580417123